entrypoint.sh

```diff
@@ -1,16 +1,46 @@
 #!/bin/sh
 
-function cleanup {
+# Function to clean up on script exit
+cleanup() {
+    echo "Unmounting mergerfs..."
     umount /merged
+    echo "Unmounted mergerfs."
 }
 
-mergerfs -o allow_other,use_ino,cache.files=partial,dropcacheonclose=true,moveonenospc=true,category.create=mfs /disks/*: /merged
+# Set debugging options
+# set -e  # Exit immediately if a command exits with a non-zero status
+# set -x  # Print commands and their arguments as they are executed
 
+# Define the default parameters file
+PARAMETERS_FILE="/config/parameters.conf"
 
+# Check if MERGERFS_PARAMS is set, override PARAMETERS_FILE if it is
+if [ -n "$MERGERFS_PARAMS" ]; then
+    echo "Using parameters from MERGERFS_PARAMS environment variable."
+    echo "$MERGERFS_PARAMS" > "$PARAMETERS_FILE"
+fi
+
+# Merge parameters from the file into a single line without comments or empty lines
```

Comment:
This is a bit convoluted given you can simply pass `-o config=$PARAMETERS_FILE` if you rework the above mechanism a bit. 
But that works I guess